jabgui/src/main/java/org/jabref/gui/frame/MainMenu.java

```diff
@@ -182,20 +183,21 @@ private void createMenu() {
                 new SeparatorMenuItem(),
 
                 // region: Sharing of the library
-
-                // TODO: Should be only enabled if not yet shared.
-                factory.createSubMenu(StandardActions.GIT,
-                        factory.createMenuItem(StandardActions.GIT_SHARE, new GitShareToGitHubAction(dialogService, stateManager))
-                ),
-
                 factory.createSubMenu(StandardActions.REMOTE_DB,
                         factory.createMenuItem(StandardActions.CONNECT_TO_SHARED_DB, new ConnectToSharedDatabaseCommand(frame, dialogService)),
                         factory.createMenuItem(StandardActions.PULL_CHANGES_FROM_SHARED_DB, new PullChangesFromSharedAction(stateManager))),
-
                 // endregion
 
                 new SeparatorMenuItem(),
 
+                // TODO: Should be only enabled if not yet shared.
+                factory.createSubMenu(StandardActions.GIT,
+                        factory.createMenuItem(StandardActions.GIT_COMMIT, new GitCommitAction(dialogService, stateManager)),
+                        new SeparatorMenuItem(),
+                        factory.createMenuItem(StandardActions.GIT_SHARE, new GitShareToGitHubAction(dialogService, stateManager))
+                ),
+                new SeparatorMenuItem(),
```

Comment:
git should really be BEFORE the remote DB - AND it is about sharing.

(Sorry for causing merge conflicts here)

Menu should really look like as follows (git first, database second)

<img width="393" height="470" alt="image" src="https://github.com/user-attachments/assets/5ef68676-55c8-434f-870e-de46ed27a03d" />